node/status_node.go

```diff
@@ -107,7 +114,30 @@ func (n *StatusNode) startWithDB(config *params.NodeConfig, db *leveldb.DB, serv
 	}
 
 	if n.discoveryEnabled() {
-		return n.startDiscovery()
+		// Check if LES is enabled before starting Discovery.
+		// If so, add LES topics to required topics.
+		if config.LightEthConfig != nil && config.LightEthConfig.Enabled {
```

Comment:
btw les code by itself uses discovery v5 server. implementation is somewhat similar to peer pool, but it never stops.